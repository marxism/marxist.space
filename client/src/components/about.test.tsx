/* eslint-env jest */

import * as React from 'react'
import About from './about'
import { create } from 'react-test-renderer'

describe('about', () => {
  test('renders', () => {
    expect(create(<About />).toJSON()).toMatchSnapshot()
  })
})
