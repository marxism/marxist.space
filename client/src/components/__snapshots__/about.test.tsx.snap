// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about renders 1`] = `
.c1 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  min-width: 0;
  margin-left: 16px;
  margin-right: 16px;
}

.c1 a {
  -webkit-transition: color 0.1s ease;
  transition: color 0.1s ease;
  text-underline-position: under;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c1 p {
  margin-bottom: 16px;
  line-height: 1.5;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  margin: 0 10vw;
}

@media (max-width:1024px) {
  .c0 {
    margin: 0 5vw;
  }
}

@media (max-width:768px) {
  .c0 {
    display: block;
    margin: 0;
  }
}

<div
  className="c0"
>
  <main
    className="c1"
  >
    <p>
      marxist.space is a
       
      <a
        href="https://prolesoft.github.io"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        ProleSoft
      </a>
       
      project. Check out our
       
      <a
        href="https://prolesoft.github.io/blog"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        developer blog
      </a>
      .
    </p>
    <p>
      This project is
       
      <a
        href="https://github.com/prolesoft/marxist.space"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        free and open source
      </a>
       
      under the
       
      <a
        href="https://github.com/prolesoft/marxist.space/blob/master/LICENSE.md"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        LGPL-3.0
      </a>
       
      license.
       
      <a
        href="https://github.com/prolesoft/marxist.space/blob/master/.github/CONTRIBUTING.md"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Contributions
      </a>
       
      are welcome. You can also email
       
      <a
        href="mailto:marxist.space@protonmail"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        marxist.space@protonmail.com
      </a>
       
      or join the discussion on
       
      <a
        href="https://matrix.to/#/!TUENHHyVTWHKRxKiSm:matrix.org?via=matrix.org"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Matrix
      </a>
    </p>
    <p>
      <a
        href="https://stats.uptimerobot.com/5Alqxc0QYg"
        rel="nofollow noopener noreferrer"
        target="_blank"
      >
        Status Page
      </a>
      .
    </p>
  </main>
</div>
`
